query-pull/src/lib.rs

```diff
@@ -143,6 +145,7 @@ pub struct Puller {
     // The range is the set of aliases to use in the output.
     attributes: BTreeMap<Entid, ValueRc<NamespacedKeyword>>,
     attribute_spec: cache::AttributeSpec,
+    db_id_alias: Option<ValueRc<NamespacedKeyword>>,
```

Comment:
Definitely wants a comment here, since this is a) not Datomic-alike and b) subtle.